imp/NonTerminals.cs

```diff
@@ -57,10 +57,304 @@ public class NonTerminals : INonTerminals {
+                               
+                               new TokenSet( /* bits: */ 0x00008000, 0x00000000, 0x00000000, /* counter: */ 1 ), /* forStatement */
+                               
+                               new TokenSet( /* bits: */ 0x00000000, 0x00000200, 0x00000000, /* counter: */ 1 ), /* designator */
+                               
+                               new TokenSet( /* bits: */ 0x00000000, 0x40000000, 0x00000044, /* counter: */ 3 ), /* selector */
+                               
+                               new TokenSet( /* bits: */ 0x01000000, 0x000C3E00, 0x00000110, /* counter: */ 10 ), /* expression */
+                               
+                               new TokenSet( /* bits: */ 0x01000000, 0x000C3E00, 0x00000110, /* counter: */ 10 ), /* simpleExpression */
+                               
+                               new TokenSet( /* bits: */ 0x01000000, 0x00003E00, 0x00000110, /* counter: */ 8 ), /* term */
+                               
+                               new TokenSet( /* bits: */ 0x01000000, 0x00003E00, 0x00000110, /* counter: */ 8 ), /* simpleTerm */
+                               
+                               new TokenSet( /* bits: */ 0x00000000, 0x00003E00, 0x00000110, /* counter: */ 7 ), /* factor */
+                               
+                               new TokenSet( /* bits: */ 0x00000000, 0x00000200, 0x00000000, /* counter: */ 1 ), /* designatorOrFuncCall */
+                               
+                               new TokenSet( /* bits: */ 0x00000000, 0x00000000, 0x00000100, /* counter: */ 1 ), /* setValue */
+                               
+                               new TokenSet( /* bits: */ 0x01000000, 0x000C3E00, 0x00000110, /* counter: */ 10 ), /* element */
+                               
+                               new TokenSet( /* bits: */ 0x00000044, 0x00000240, 0x00000000, /* counter: */ 4 ), /* formalType */
+                               
+                               new TokenSet( /* bits: */ 0x00000040, 0x00000040, 0x00000000, /* counter: */ 2 ), /* attributedFormalType */
+                               
+                               new TokenSet( /* bits: */ 0x00000040, 0x00000240, 0x00000000, /* counter: */ 3 ), /* formalParamList */
+                               
+                               new TokenSet( /* bits: */ 0x00000040, 0x00000240, 0x00000000, /* counter: */ 3 ), /* formalParams */
+                               
+                               new TokenSet( /* bits: */ 0x00000040, 0x00000040, 0x00000000, /* counter: */ 2 ), /* attribFormalParams */
+                               
+                               new TokenSet( /* bits: */ 0x58000004, 0x00000242, 0x00000050, /* counter: */ 9 ), /* typeDeclarationTail */
+                           };
+    #endregion
 

```

Comment:
Please remove all the `//SAM CHANGE` comments. They have no documentary value for users and maintainers. For any temporary comments during review, we can use Github annotation features.